css/custom.scss

```
@@ -330,4 +330,37 @@ $link-active: $link-hover;
 
 .peoples {
   margin-top: 1em;
-}
\ No newline at end of file
+}
+
+.button {
```

```suggestion
.button, a.button, .content a.button {
```
one of these rules should help override the grey text that's hard to read